File: OLSource1/v1/VS_officedev/codesnippet/CSharp/accessing-a-form-region-at-run-time_1.cs
        private void Access_Form_Regions_ByInspector()
        {
            WindowFormRegionCollection formRegions = 
                Globals.FormRegions
                    [Globals.ThisAddIn.Application.ActiveInspector()];
            formRegions.FormRegion1.textBox1.Text = "Hello World";
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/accessing-a-form-region-at-run-time_1.vb
    Private Sub Access_Form_Regions_ByInspector()
        Dim formRegions As WindowFormRegionCollection = Globals.FormRegions _
            (Globals.ThisAddIn.Application.ActiveInspector())
        formRegions.FormRegion1.textBox1.Text = "Hello World"
    End Sub
